### ISSUE TITLE
Exception thrown causing incomplete output to console 

### ISSUE THREAD

**Issue by danielkcz** [remy/nodemon#311]
Alright, I am not sure if this isn't purely Windows issue and I don't have option to test anywhere else. Given this really simple code:

```
console.log('Calling console.log');
console.error('Calling console.error');
console.log('Calling console.log second time - fails');
console.error('Calling console.error second time - fails');
nonexisting.throwError();
```

When I run it directly like `node server.js` it works just fine, all 4 messages are displayed followed by exception details. However running `nodemon server.js` gives me just this:

```
16 Mar 20:38:56 - [nodemon] starting `node server.js`
Calling console.log
Calling console.error

S:\workspace\server.js:5
nonexisting.throwError();
^
ReferenceError: nonexisting is not defined
......
16 Mar 20:38:56 - [nodemon] app crashed - waiting for file changes before starting...
```

What could be possibly causing this ? It doesn't make any sense... Of course commenting out that last error line displays all messages correctly.

It would not be that big deal, but thing is, that adding Coffeescript to the mix with  `nodemon server.coffee` swallows whole exception too (maybe worth another issue submit). I am trying to utilize [debug utility](https://github.com/visionmedia/debug) to have nice verbose log of what is happening in the app. Currently everything is just blind with no clue where the error occurred.

Oh and adding `process.on('uncaughtException', ...` somewhat fixes JS version, but Coffee version is still broken same way :( Not that it's solution I like, I want to handle exceptions on my own, but I am talking mainly about development times here and when something slips out.
